TYPO3.Media/Classes/TYPO3/Media/Controller/AssetController.php

```diff
@@ -259,6 +268,42 @@ public function newAction()
     }
 
     /**
+     * @param Asset $asset
+     * @return void
+     */
+    public function replaceAssetResourceAction(Asset $asset)
+    {
+        $maximumFileUploadSize = $this->maximumFileUploadSize();
+        $this->view->assignMultiple(array(
+            'asset' => $asset,
+            'maximumFileUploadSize' => $maximumFileUploadSize,
+            'redirectPackageEnabled' => $this->packageManager->isPackageAvailable('Neos.RedirectHandler'),
+            'humanReadableMaximumFileUploadSize' => Files::bytesToSizeString($maximumFileUploadSize)
+        ));
+    }
+
+    /**
+     * Replace the resource on an asset.
+     *
+     * @param AssetInterface $asset
+     * @param FlowResource $resource
+     * @param array $options
+     * @return void
+     */
+    public function updateResourceAction(AssetInterface $asset, FlowResource $resource, array $options = [])
```

Comment:
I find it a little inconsistent this is named `updateResource` when the other one is named `replaceAssetResource`